src/main.rs

```diff
@@ -191,6 +194,16 @@ fn main() {
                 .parse()
                 .expect("Number of threads should be a number");
             i += 1;
+        } else if args[i] == "-o"{
+            if args.len() <= i + 1{
+                eprintln!("[ERROR]: Output file not specified.\n");
+                print_usage(&args[0]);
+                process::exit(1);
+            }
+            if &args[i + 1] != "-"{
```

Comment:
You can now remove this check

Reply:
So to confirm, I shouldn't check for the special case "-o -" to write to stdout?